Library/Homebrew/cmd/update-bash.sh

```diff
@@ -208,6 +208,12 @@ update-bash() {
     esac
   done
 
+  if ! curl -I --connect-timeout 2 https://github.com &> /dev/null
```

Comment:
Maybe just worth using `ping`: `ping -c5 -t1 github.com` or `ping -c5 -t1 8.8.8.8` to avoid DNS lookup and go for a Google server?